int8 idHello = 0
int8 idPointAt = 1

int8 gestureId
geometry_msgs/Point ground_point # Only used when gesture is PointAt
